storage/storage_tests/create_country_info_getter.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "geometry/mercator.hpp"
+
+#include "std/unique_ptr.hpp"
+
+namespace storage
+{
+class CountryInfoGetter;
+
+unique_ptr<CountryInfoGetter> CreateCountryInfoGetter();
```

Comment:
Your example takes place only once. But anyway if you insist I'll use variant with one function.